Xamarin.Forms.Build.Tasks/XamlCTask.cs

```diff
@@ -44,24 +48,29 @@ public override bool Execute(out IList<Exception> thrownExceptions)
 				return true;
 			}
 
-			var resolver = new XamlCAssemblyResolver();
-			if (!string.IsNullOrEmpty(DependencyPaths))
+			var resolver = DefaultAssemblyResolver ?? new XamlCAssemblyResolver();
+			var xamlCResolver = resolver as XamlCAssemblyResolver;
+
+			if (xamlCResolver != null)
```

Comment:
For me it's not a warning as most resolver won't support this. However I do see your point that we are ignoring properties at that point. I'm happy either way.